### PR TITLE
SDIT-994 Add tests for duplicate mapping and set up delete event ready for synchronisation

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/nonassociations/NonAssociationsSynchronisationService.kt [hunk 1]
```diff
@@ -76,6 +76,61 @@ class NonAssociationsSynchronisationService(
...
+        event.toTelemetryProperties(),
+      )
+      return
+    }
+    */
+
+    telemetryClient.trackEvent(
+      "non-association-delete-synchronisation-success",
+      event.toTelemetryProperties(),
+    )
```
Comment (karenmillermoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/nonassociations/NonAssociationsSynchronisationService.kt:101]: Have just moved telemetry to here, ready for adding delete code

**Review by steverendell — APPROVED** [review]
lgtm
